django_irods/views.py

```diff
@@ -228,7 +228,7 @@ def download(request, path, use_async=True, use_reverse_proxy=True,
                 user_id = get_task_user_id(request)
                 if not task_id:
                     # create the bag
-                    task = create_bag_by_irods.apply_async((res_id, user_id))
+                    task = create_bag_by_irods.apply_async((res_id, ))
```

Comment:
can the comma after ```res_id``` be removed? I have detailed explanation for this comment in my slack message.